packages/tpetra/core/src/Tpetra_Details_checkLaunchBlocking.hpp

```diff
@@ -17,7 +17,7 @@
 
 namespace Tpetra {
 namespace Details {
-#ifdef HAVE_TPETRACORE_CUDA
+#if defined(HAVE_TPETRACORE_CUDA) && defined(KOKKOS_ENABLE_DEPRECATED_CODE_4)
```

Comment:
Out of curiosity, what part of this relies on deprecated Kokkos features? If it's the old architectures, Kokkos still supports cuda 11 which fully supports maxwell.

Reply:
@brian-kelley 
The use of `Kokkos::Cuda::device_arch()` was deprecated in https://github.com/kokkos/kokkos/pull/6710 , which seemed to be the key component to the routines functionality

Reply:
OK, that makes sense.